user/hare/files/config.mk

```diff
@@ -0,0 +1,43 @@
+# install locations
+PREFIX = /usr/local
+BINDIR = $(PREFIX)/bin
+MANDIR = $(PREFIX)/share/man
+SRCDIR = $(PREFIX)/src
+STDLIB = $(SRCDIR)/hare/stdlib
```

Comment:
i could get rid of the variables here that get overriden by cbuild anyway, if you want (or maybe just comment them out)

Reply:
seems fine as is